pyproject.toml

```diff
@@ -84,9 +84,6 @@ test-command = "pytest --ignore {project}/tests/plotting --ignore {project}/test
 skip = ["pp*", "*musllinux*"]
 build-verbosity = 1
 
-[tool.cibuildwheel.macos]
-archs = ["x86_64", "arm64"]
```

Comment:
How is this defined now? Are those both built by default?

Reply:
Just by using a macos-13 and macos-14 runner, it will build on native archs.